src/main/io/serial.c

```diff
@@ -293,6 +302,21 @@ serialPort_t *openSerialPort(
             serialPort = uartOpen(USART3, callback, baudRate, mode, options);
             break;
 #endif
+#ifdef USE_USART4
+    case SERIAL_PORT_USART4:
+        serialPort = uartOpen(UART4, callback, baudRate, mode, options);
```

Comment:
the rename would go for uartOpen etc as well, the driver class is already called serial so something like serialOpen(..) would make more sense, no?